### PR TITLE
new tags

### PR DESCRIPTION
**Pull request by Paolaufer**
…te', 'mendix-salesorders' from QA to Production

### CODE REVIEW

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Create an SAP Fiori App Using SAP Business Application Studio
 description: Develop a simple SAP Fiori application that will present a list of suppliers from an on-premise backend.
 auto_validation: true
 time: 25
-tags: [ tutorial>beginner, topic>sapui5, products>sap-fiori, products>sap-business-technology-platform, products>sap-btp--cloud-foundry-environment]
+tags: [ tutorial>beginner, topic>sapui5, products>sap-fiori, products>sap-cloud-platform, products>sap-cloud-platform-for-the-cloud-foundry-environment]
```
Comment (MichaelCzcz) [inline tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:6]: Why did you change the correct new tags back to the old invalid versions?

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md (hunk 1 above)
Comment (Paolaufer) [reply tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:6]: Not intentionally, I had problems pushing to Prod and was asked to reset the files. I must have copy/pasted from the wrong file.
I'll change them again.

**Comment by Paolaufer** [comment]
added the new tags

File: tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: Create an SAP Fiori App Using SAP Business Application Studio
 description: Develop a simple SAP Fiori application that will present a list of suppliers from an on-premise backend.
 auto_validation: true
 time: 25
-tags: [ tutorial>beginner, topic>sapui5, products>sap-fiori, products>sap-business-technology-platform, products>sap-btp--cloud-foundry-environment]
+tags: [ tutorial>beginner, topic>sapui5, products>sap-fiori, products>sap-business-technology-platform, products>sap-btp-for-the-cloud-foundry-environment]
```
Comment (MichaelCzcz) [inline tutorials/appstudio-fioriapps-create/appstudio-fioriapps-create.md:6]: products>sap-btp--cloud-foundry-environment is the correct tag.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
